CONTRIBUTOR_LADDER.md

```diff
@@ -0,0 +1,161 @@
+    * Entitled to vote in the [TODO: appropriate election]
+    * Can be added to GitHub teams in the Open Cluster Management org
+    * Can recommend other contributors to become Org Member
+
+The process for a Contributor to become an Organization Member is as follows:
+
+<!-- the process of becoming an organization member is going to depend strongly on how your project manages its infrastructure. TODO: Project leads to fill in exact details of how a contributor becomes an organization member-->
+1. The contributor or an Org Member may propose them for membership in an Issue in the relevant Subproject.  This issue will name at least two existing Org Members (sponsors) who can vouch for the contributor, and will list their existing contributions.
+2. Both sponsors will +1 the request.
+3. GitHub administrator will add the contributor to the GitHub organization, as well as to the relevant Subproject's general contributor Team.
+
+### Reviewer
+<!-- Some projects have CI/CD systems that allow for designating people as official reviewers, whose reviews count towards a PR being accepted into the project.  Other projects offer reviewers specific recognition and status.  This role is for either of those kinds of projects. Smaller projects will not use it.-->
+<!--TODO: project leads to fill in exact details of this role for your project-->
+Description: A Reviewer has responsibility for specific code, documentation, test, or other project areas. They are collectively responsible, with other Reviewers, for reviewing all changes to those areas and indicating whether those changes are ready to merge. They have a track record of contribution and review in the project.
+
+Reviewers are responsible for a "specific area." This can be a specific code directory, driver, chapter of the docs, test job, event, or other clearly-defined project component that is smaller than an entire repository or subproject. Most often it is one or a set of directories in one or more Git repositories. The "specific area" below refers to this area of responsibility.
+
+Reviewers have all the rights and responsibilities of an Organization Member, plus:
+
+* Responsibilities include:
+    * Following the reviewing guide
+    * Reviewing most Pull Requests against their specific areas of responsibility
+    * Reviewing at least 10 PRs per year
+    * Helping other contributors become reviewers
+* Requirements:
+    * Experience as a Contributor for at least 3 months
+    * Is an Organization Member
+    * Has reviewed, or helped review, at least 10 Pull Requests
+    * Has analyzed and resolved test failures in their specific area
+    * Has demonstrated an in-depth knowledge of the specific area
+    * Commits to being responsible for that specific area
+    * Is supportive of new and occasional contributors and helps get useful PRs in shape to commit
+* Additional privileges:
+    * Has GitHub or CI/CD rights to approve pull requests in specific directories
```

Comment:
At least some of the open-cluster-management repos have separate lgtm and approve. Is it intentional that this Reviewer role holds approving power?